labs/lab08-airgap-lite/README.md

```diff
@@ -114,134 +109,132 @@ curl -v https://kubernetes.io
-Replace the URL with the one you copied above.
+4. SSH to your jump box (the one with the public IP) `ssh kots@<jump box IP address>` and check the download of the Kubernetes installer (kURL) bundle.
 
-At the beginning of the lab, we downloaded the bundle with this command from the Jump box.
+5. Replace the URL with the one you copied above.
+
+At the beginning of the lab, we downloaded the bundle with this command from the Jump box:
 
 ```text
 kots@dx411-dex-lab08-airgap-lite-jump ~$ curl -o kurlbundle.tar.gz <URL>
 ```
 
-It should be finished now, so you can copy it to the Air Gap server. 
-You can use the DNS name in this case, as described in [Instance Overview](#instance-overview).
+It should be finished now, so you can copy it to the Air Gap server.
+You can use the DNS name in this lab, as described in the [Instance Overview](#instance-overview).
 
 ```text
 kots@dx411-dex-lab08-airgap-lite-jump ~$ scp kurlbundle.tar.gz ${REPLICATED_APP}-lab08-airgap-lite:~
 
 ```
 
-> **Note**: -- we use SCP via an SSH tunnel in this case, but the Air Gap methods in this lab also extend to
-more locked down environments where e.g. physical media is required to move assets into the datacenter.
+> **Note**: -- we use SCP via an SSH tunnel in this lab, but the Air Gap methods in this lab also extend to more locked down environments where, for example, physical media is required to move assets into the datacenter.
 
-Now we'll SSH all the way to Air Gap node. If you still have a shell on your jump box, you can use the instance name.
+The next step is to SSH all the way to Air Gap node. If you still have a shell on your jump box, you can use the instance name:
 
 ```text
 kots@dx411-dex-lab08-airgap-lite-jump ~$ ssh ${REPLICATED_APP}-lab08-airgap-lite
 ```
 
-Otherwise, you can use the one below 
+Otherwise, you can use the one below:
 
 ```shell
 ssh -J ${FIRST_NAME}@${JUMP_BOX_IP} ${FIRST_NAME}@${REPLICATED_APP}-lab08-airgap-lite
 ```
 
-Once you're on the Air Gap node, untar the bundle and run the install script with the `airgap` flag.
-kURL install flags are documented [in the kurl.sh docs](https://kurl.sh/docs/install-with-kurl/advanced-options).
+When you are on the Air Gap node, untar the bundle and run the install script with the `airgap` flag.
+The Kubernetes installer (kURL) install flags are documented [in the kurl.sh docs](https://kurl.sh/docs/install-with-kurl/advanced-options).
 
 ```shell
 tar xvf kurlbundle.tar.gz
 sudo bash install.sh airgap
 ```
 
-At the end, you should see a `Installation Complete` message as shown below. Since the instance is Air Gap, we'll need to create a port forward to access the UI from your workstation in the next step.
+At the end, you should see an `Installation Complete` message. Because this instance is an Air Gap, you must create a port forward to access the UI from your workstation in the next step.
```

Comment:
```suggestion
At the end, you should see an `Installation Complete` message. Because this instance is in Air Gap mode, you must create a port forward to access the UI from your workstation in the next step.
```
`an Air Gap` sounds weird? Or just me?